app/views/search/_master_overlay.html.erb

```diff
@@ -0,0 +1,140 @@
+<div class="master-overlay homepage-search is-master-overlay-secondary-hidden is-master-overlay-maximized" id="collections-overlay">
+  <%= render partial: 'overlay_parent' %>
+  <div class="master-overlay-main">
+    <div class="master-overlay-main-content" data-level="0">
+      <section id="collection-results" class="master-overlay-has-min" data-master-overlay-title="Collections">
+        <a class="master-overlay-minimize master-overlay-when-max" href="#" title="Minimize"><i class="fa fa-angle-double-left"></i></a>
+        <a class="master-overlay-maximize master-overlay-when-min" href="#" title="Maximize"><i class="fa fa-angle-double-right"></i></a>
+        <nav class="master-overlay-nav master-overlay-when-parent-hidden">
+          <a href="#" class="master-overlay-show-parent" data-bind="click: showParent">
+            <i class="fa fa-search"></i> Browse Collections
+            <span class="master-overlay-when-max" data-bind="if: query.facets() && query.facets().length > 0">
+               (<span data-bind="text: pluralize(query.facets().length, 'facet', 'facets')"></span> applied)
+            </span>
+          </a>
+        </nav>
+        <header>
+          <h2><strong class="collection-hits" data-bind="text: collections.hits"></strong> Matching Collections <a class="master-overlay-hide-parent master-overlay-show-next master-overlay-forward low-priority" href="#"><i class="fa fa-question-circle"></i></a></h2>
+        </header>
+        <div class="master-overlay-info master-overlay-when-min">
+          <p data-bind="visible: !project.isEmpty()" class="button-details">
+            <a id="view-project-min" class="button master-overlay-hide-parent master-overlay-show-next master-overlay-forward" href="#">
+              <i class="fa fa-folder-o"></i> View Project
+            </a>
+          </p>
+        </div>
+        <div class="master-overlay-info master-overlay-when-max">
+          <div class="collection-filters">
+            <input type="checkbox" id='has-granules' data-bind="checked: query.hasGranules" />
+            <label for="has-granules">Hide collections without granules (<!-- ko text: '23' --><!-- /ko -->)</label>
```

Comment:
Wrote CMR-3411 just now. Hopefully they can come up an idea to give us this piece of info. Otherwise, we will have to make two calls to CMR and do a subtraction to get the number.